sql/moz-fx-data-shared-prod/marketing_campaign_names/asa_v1/udf.sql

```diff
@@ -0,0 +1,53 @@
+CREATE OR REPLACE FUNCTION marketing_campaign_names.asa_v1(campaign_name STRING)
```

Comment:
I put these UDFs in shared-prod because I don't expect anyone to use them directly. I'm adding a helper function in mozfun that will use the proper function based on the ad network / version, but I need these deployed first (I cannot create the dataset `marketing_campaign_names` myself).

Open to changing the dataset name.

Reply:
These might be the only udf.sql that aren't in `udf`, `udf_js`, or `mozfun` so I'm wondering if there's any bqetl code that makes that assumption.  I'm mostly sure that's not the case but something to keep in mind in case you see weird behaviour

Reply:
Oh that's a good point, I didn't realize it at the time. Mostly the tests passed, I'm wondering if these will get deployed correctly (I suppose I could have checked in my sandbox project)